content/javascript/concepts/arrays/terms/every/every.md

```diff
@@ -0,0 +1,73 @@
+---
+Title: '.every()'
+Description: 'Checks if all elements in an array satisfy the condition specified by the given function.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Methods'
+  - 'Iterators'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`.every()`** method iterates over each element in an array and stops when a falsy value is returned by the provided callback function. If a falsy value is found, every() returns `false`. Otherwise, if the callback function returns a truthy value for all elements, every() returns `true`.
```

Comment:
```suggestion
The **`.every()`** method iterates over each element in an array and stops when a falsy value is returned by the provided callback function. If a falsy value is found, `.every()` returns `false`. Otherwise, the function will return `true`.
```